### ISSUE TITLE
Histogram won't show in Tensorboard

### ISSUE THREAD

**Issue by jziegle4** [Unity-Technologies/ml-agents#5403]
When using StatAggregationMethod.Histogram data recorder with Academy.Instance.StatsRecorder it will not show in Tensorboard, nether in scalar nor in histograms

```
m_Recorder = Academy.Instance.StatsRecorder 
m_Recorder.Add("Value", value, StatAggregationMethod.Histogram)

```
No Console Output or Exceptions

- Unity Version: 2021.1.4f1
- OS + version: Windows 10
- _ML-Agents version_: 0.26.0
- _Torch version_: 1.8.1+cu111
- _Environment_: recreated with 3dBall

**Comment by vincentpierre** [comment]
Hi @jziegle4 
I was able to reproduce this issue. I have a potential fix here #5410
Let us know if it resolves your issue.

**Comment by jziegle4** [comment]
Hi @vincentpierre 

Thank you, this resolves the issue!
